src/core/border_router/routing_manager.cpp

```diff
@@ -755,7 +755,7 @@ void RoutingManager::EvaluateRoutingPolicy(void)
             nextSendDelay = kMaxInitRtrAdvInterval;
         }
 
-        LogInfo("Router advertisement scheduled in %u seconds", nextSendDelay);
+        LogInfo("Routing policy evaluation scheduled in %u seconds", nextSendDelay);
```

Comment:
Maybe we can simply remove this log because we already have similar log in `StartRoutingPolicyEvaluationDelay`?

Reply:
Goodpoint. Done.